### PR TITLE
fixed photo style and moment.format

### PR DESCRIPTION
**Pull request by imhyojunglee**
1. fixed photo style 
2. change moment.format

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -6,18 +6,20 @@ function Footer(props) {
...
         <footer className={classes.footer}>
-            <p>Developed     
-                <a href="#">Eric Kim</a>
-                <a href="#">ABC</a>
-                <a href="#">123</a>
-                <a href="#">DDD</a>
-                <a href="#">event_date</a>
+            <p>
+                Developed
+                <a href="//www.linkedin.com/company/vancouver-kdd/">Eric Kim</a>
```
Comment (Naricky) [inline src/components/Footer.js:11]: this is very nit-picky, but could we iterate through these <a> tags. Maybe create a const obj with these values to iterate -? ( We should get these data from backend ) More so to ensure when this is used as your portfolio, to demonstrate your knowledge of jsx looping

File: src/pages/Photospage.js [hunk 2]
```diff
@@ -51,11 +50,10 @@ const Photospage = () => {
...
                                 <>
-                                    {!!index||<h2>{moment(photo.created_at).format('LL')}</h2>}
-                                    {moment(moment(photo.created_at).format('LL')).isSame(
-                                        moment(photos[Math.max(index - 1, 0)].created_at).format('LL'),
-                                        'day',
-                                    ) || <h2>{moment(photo.created_at).format('LL')}</h2>}
+                                    {!!index || <h2>{moment(photo.created_at).format('LL')}</h2>}
+                                    {moment(photo.created_at).isSame(moment(photos[Math.max(index - 1, 0)].created_at), 'day') || (
+                                        <h2>{moment(photo.created_at).format('LL')}</h2>
+                                    )}
```
[the remark this answers is not in the document]
Comment (bosung90) [reply src/pages/Photospage.js:56]: Created task on Clickup https://app.clickup.com/t/1e6e61w

File: src/components/Footer.js (hunk 1 above)
Comment (bosung90) [reply src/components/Footer.js:11]: Created task on Clickup https://app.clickup.com/t/1e6e66d

**Review by bosung90 — APPROVED** [review]
There were 2 comments but since they are not a must fix at this point, I have created 2 separate tasks on clickup so they can be done on a different PR.

**Pull request merged**